backend/app/Http/Controllers/LearnedWordsController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class LearnedWordsController extends Controller {
+    public function index() {
+        $id = Auth::id();
+
+        $user = User::find($id);
```

Comment:
Laravel provides also to retrieve the authenticated user.
Ref: https://laravel.com/docs/7.x/authentication#retrieving-the-authenticated-user
```suggestion
        $user = Auth::user();
```

Reply:
ahhh for this one sir, i need the User Model itself not the Authenticable User

![image](https://user-images.githubusercontent.com/95029803/149860154-b1145602-f528-4de9-9e73-faf7239059b5.png)

Reply:
Yet on your line 11, you are retrieving the Authenticated Id.
Auth::user() should return a User instance.

Reply:
yes sir, Auth::user() returns 

![image](https://user-images.githubusercontent.com/95029803/149861058-7d4b9ed6-dae9-4e08-b121-0f3cee6d7cd7.png)

Authenticable Class

Whereas i need 
`App\Models\User;`

to access the User model `quiz_logs` function

Reply:
ahhh the relationship, i see.

Reply:
> ahhh for this one sir, i need the User Model itself not the Authenticable User
> 
> ![image](https://user-images.githubusercontent.com/95029803/149860154-b1145602-f528-4de9-9e73-faf7239059b5.png)

here sir, nag eerror kasi hindi Models\User